docs/start/introduction.md

```diff
@@ -0,0 +1,84 @@
+# A Friendly Introduction
```

Comment:
change the name to tutorial

Reply:
But here I do not cover how to use the package